src/yup-phone.ts

```diff
@@ -46,6 +46,9 @@ Yup.addMethod(Yup.string, YUP_PHONE_METHOD, function yupPhone(
       strict = false;
     }
 
+    // This is what .required() is for
+    if (!value || ((typeof value === 'string') && value.length === 0)) return true
```

Comment:
It would be nice be explicitly about `!value` i.e.
```suggestion
    if (value == null || ((typeof value === 'string') && value.length === 0)) return true
```